Server/copy-assets.ts

```diff
@@ -0,0 +1,37 @@
+import * as path from 'path';
```

Comment:
Please put this change into its own pr.

Reply:
will split into separate PR

Reply:
separate PR https://github.com/citrineos/citrineos-core/pull/209